src/enkf/gridinfo_nmmb.f90

```diff
@@ -25,6 +25,8 @@ module gridinfo
                                                                                 'cw', 'prse', 'ql', 'qr', 'qi',    &
                                                                                 'qli', 'dbz', 'w'/)
 character(len=max_varname_length),public, dimension(2) :: vars2d_supported = (/ 'ps', 'sst' /)
```

Comment:
Not related to this PR. But I am curious, here (and gridinfo_wrf.90) the dimension of vars2d_supported is 2, however, it is 3 in gridinfo_fv3reg.f90. 
https://github.com/NOAA-EMC/GSI/pull/705/files#diff-a41d072f43b85f6a5068c61b92effaf58095c81e9a6bd6ea63734389c6e0766aR79-R81